### PR TITLE
QMR - Update Serverless Framework to V3

### PR DESCRIPTION
**Pull request by jdinh8124**
## Purpose
QMR - Update Serverless Framework to V3
https://qmacbis.atlassian.net/browse/OY2-17283

#### Linked Issues to Close
https://qmacbis.atlassian.net/jira/software/c/projects/OY2/boards/208?modal=detail&selectedIssue=OY2-17838

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: services/.sechub/serverless.yml [hunk 1]
```diff
@@ -21,20 +21,20 @@ custom:
       - master
       - val
       - prod
-  githubAccessToken: ${ssm:/configuration/${self:custom.stage}/sechub/githubAccessToken~true, ssm:/configuration/default/sechub/githubAccessToken~true}
-  githubRepository: ${ssm:/configuration/${self:custom.stage}/sechub/githubRepository~true, ssm:/configuration/default/sechub/githubRepository~true, env:GITHUB_REPOSITORY}
-  githubRepositoryProjects: ${ssm:/configuration/${self:custom.stage}/sechub/githubRepositoryProjects~true, ssm:/configuration/default/sechub/githubRepositoryProjects~true, ""}
-  githubOrganizationProjects: ${ssm:/configuration/${self:custom.stage}/sechub/githubOrganizationProjects~true, ssm:/configuration/default/sechub/githubOrganizationProjects~true, ""}
-  severity: ${ssm:/configuration/${self:custom.stage}/sechub/severity~true, ssm:/configuration/default/sechub/severity~true, "CRITICAL,HIGH"}
```
Comment (jphechter) [inline services/.sechub/serverless.yml:28]: I'm not super familiar with all the Serverless docs, but it looks as though `~true` is required to decrypt a secure string. Is this something that is no longer required in version 3 or is this related to a change we made in AWS?

This comment is related to every similar change where we're dropping `~true`

**Review by jphechter — COMMENTED** [review]
This looks good, mostly curious about the implementation

File: services/.sechub/serverless.yml (hunk 1 above)
Comment (abelzer212) [reply services/.sechub/serverless.yml:28]: the ~true terminology has been deprecated with this version.

**Review by abelzer212 — APPROVED** [review]
This looks great, everything is still working and our deploy logs are SO short now

**Review by jphechter — DISMISSED** [review]

**Review by abelzer212 — APPROVED** [review]
